docs/installation.md

```diff
@@ -4,16 +4,35 @@ title: Installation
 
 It is strongly recommended that you use a released version of dep. While tip is never purposefully broken, its stability is not guaranteed.
 
-Pre-compiled binaries are available on the [releases](https://github.com/golang/dep/releases) page. On MacOS, you can also install or upgrade to the latest released version with Homebrew:
+
+## Manual Installation
+
+Pre-compiled binaries are available on the [releases](https://github.com/golang/dep/releases) page.
+
+## MacOS
```

Comment:
`###`?

Reply:
This should be a subheader under `Manual Installation`, right?

Reply:
My thinking here was "Manual installation" being just downloading and running the binaries vs. using a package manager. Would changing the "Manual Installation" header make more sense?

Reply:
Oh, I misunderstood the header. They are all technically manual installations.

I think `Binary Install` might a better header. What do you think?